glean_parser/swift.py

```diff
@@ -149,31 +161,32 @@ def output_swift(
     namespace = options.get("namespace", "GleanMetrics")
     glean_namespace = options.get("glean_namespace", "Glean")
 
-    for category_key, category_val in objs.items():
-        filename = util.Camelize(category_key) + ".swift"
-        filepath = output_dir / filename
+    filename = "Metrics.swift"
+    filepath = output_dir / filename
+    metrics = []
 
+    for category_key, category_val in objs.items():
         custom_pings = defaultdict()  # type: Dict[str, pings.Ping]
         for obj in category_val.values():
             if isinstance(obj, pings.Ping):
                 custom_pings[obj.name] = obj
```

Comment:
Maybe just `contains_pings = any(isinstance(obj, pings.Ping) for obj in category_val.values())`?

Reply:
Hm, makes sense.